CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Removed
 
+- The 'clearVisitedNodes' parameter of Dialogue.UnloadAll
```

Comment:
This is an internal code change, so it doesn't affect users. I'm fine to skip having this change be visible in the change log.